src/stdlib.js

```diff
@@ -99,7 +99,7 @@ const unstableGlobalPropertyNames = [
   'Intl'
 ];
 
-export function getSharedGlobalDescs(unsafeGlobal) {
+export function getSharedGlobalDescs(unsafeGlobal, deoptimize = false) {
```

Comment:
I think that in the context of the shim "deoptimize" is ambiguous. The "optimizer" already refers to another part of the code, the creation and injection of a `const` declaration. In fact "deoptimize" should mean turning off the optimization, that portion of the code:
https://github.com/Agoric/realms-shim/blob/master/src/optimizer.js#L99

Instead, what this new option really is to make the globals mutable (and _after_ the optimizer detects its and turns itself off). But if you later freeze/harden the global object, then the optimizer still is activated, hence the confusion.

We should call this option something like `writableGlobalProperties`. That best describe what is actually enabled here.

Reply:
How about `writableStableGlobals`, since it doesn't affect other global properties?

Reply:
Yes, I think it's much better. But you see the point. Maybe we can make all globals mutable. I don't think there is anything special about the "stable" globals. I would make them all mutable, the three constants because they are resolved on the global object:
```js
with (new Proxy({}, { has: (t, p) => console.log(p) })) { undefined }
with (new Proxy({}, { has: (t, p) => console.log(p) })) { NaN }
with (new Proxy({}, { has: (t, p) => console.log(p) })) { Infinity }
``` 
Outputs:
```
undefined
NaN
Infinity
```

Reply:
Okay, I've tried that with and without the `configurableGlobals` flag and all tests pass.